sdks/unreal/Agones/README.md

```diff
@@ -0,0 +1,65 @@
+# Agones UE4 Plugin
+
+The Agones [plugin](https://docs.unrealengine.com/en-US/Programming/Plugins/index.html)
+for UE4 communicates lifecycle events to the Agones sidecar.
+
+The plugin requires UE4 4.23 or later.
+
+## Installation
+
+Copy the *Agones* folder containing this README to one of the UE4 [Plugin

```

Comment:
Has this been explicitly tested as both a Project Plugin and an Engine Plugin? I ask because there are different compile time checks for these. Making your plugin Engine Plugin compatible ensures it is both Engine and Project Plugin supported.

Reply:
Not that I know of. I'll follow @markmandel's suggestion and just link to the docs instead of having a separate set of installation instructions here.